front/app/containers/Admin/projects/components/ProjectRow/index.tsx

```diff
@@ -43,23 +43,25 @@ type CustomButtonAction = {
   icon: IconNames;
   processing?: boolean;
 };
-type ButtenAction = CustomButtonAction | 'manage' | 'delete';
+type ButtonAction = CustomButtonAction | 'manage';
 
 interface Props {
   publication: IAdminPublicationContent;
-  actions?: ButtenAction[];
+  actions: ButtonAction[];
   hidePublicationStatusLabel?: boolean;
   className?: string;
+  showMoreActions?: boolean;
 }
 
 const ProjectRow = ({
   publication,
   actions,
   hidePublicationStatusLabel,
   className,
+  showMoreActions = false,
 }: Props) => {
   const [isBeingDeleted, setIsBeingDeleted] = useState<boolean>(false);
```

Comment:
Currently, `setIsBeingDeleted` is not used and so CI fails.

It will be properly handled here https://citizenlab.atlassian.net/browse/CL-2767